packages/valory/agents/oracle/aea-config.yaml

```diff
@@ -116,3 +116,4 @@ config:
       address: ${LEDGER_ADDRESS:str:http://localhost:8545}
       chain_id: ${LEDGER_CHAIN_ID:int:31337}
       poa_chain: ${LEDGER_POA_CHAIN:bool:false}
+      default_gas_price_strategy: ${LEDGER_DEFAULT_GAS_PRICE_STRATEGY:str:eip1559_polygon}
```

Comment:
Not too sure if we should change this to be be the default pricing mechanism. We can instead do an override within the service spec to enable this?

Reply:
Yes default should be what we were using so far!